.github/workflows/test_chart.yaml

```diff
@@ -0,0 +1,46 @@
+name: Lint and Test Charts
+
+on: pull_request
+
+env:
+  TAG: dev
+  IMAGE: controller
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.12.1
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.4.0
+
+      - name: setupGo
+        uses: actions/setup-go@v4
+        with:
+          go-version: '=1.20.4'
```

Comment:
let's use `1.20.6` that is the version we are trying to [use](https://github.com/rancher-sandbox/rancher-turtles/pull/51/files#diff-d9a956120ac84289d2650137f64bd8f0893331de05e44cc41899dd984c9e0d48R22) in GH workflows